### ISSUE TITLE
TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string

### ISSUE THREAD

**Issue by xeleniumz** [sequelize/sequelize-auto#213]
I can't generate model with sequelize-auto. How to fix it ?

node version - 9.5.0
databse - Mysql

**Comment by xeleniumz** [comment]
I solved this problem by edit file /bin/sequelize-auto

change

```
if (argv.c) {
  configFile = require(path.resolve(argv.c));
}
```
to
`  configFile = require(path.resolve(argv.c));`

but i think this isn't the right point to fix this error.

**Comment by steveschmitt** [comment]
Fixed in `sequelize-auto@0.5.3`
